src/inmanta/protocol/methods.py

```diff
@@ -50,6 +50,8 @@ async def add_env(env: uuid.UUID, metadata: dict) -> uuid.UUID:
 async def ignore_env(obj: Any, metadata: dict) -> Any:
     """
     This mapper only adds an env all for authz
+
+    :meta private:
```

Comment:
This is to hide this method from the public docs

Reply:
I see. I think that might be a good idea. The docstring is probably the easiest.